code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -16,6 +16,9 @@ datum
 		var/list/secondary_results = list()		//additional reagents produced by the reaction
 		var/requires_heating = 0
 
+		var/required_temp = 0

```

Comment:
the highest it can feasibly go with current systems aside from badminning is actually 1000K